src/jllvm/class/ByteCodeIterator.cpp

```diff
@@ -94,6 +94,16 @@ requires(std::is_same_v<OpCode, SIPush>) ByteCodeOp parseSIPush(const char* byte
     return SIPush{offset, consume<std::int16_t>(bytes)};
 }
 
+template <class OpCode>
```

Comment:
You ought to be able to write `template <std::same_as<MultiANewArra> OpCode>` instead of the `requires`